packages/atomic/src/components/search/facets/atomic-automatic-facet-builder/atomic-automatic-facet-builder.tsx

```diff
@@ -91,6 +93,11 @@ export class AtomicAutomaticFacetBuilder implements InitializableComponent {
         );
       }
     );
+    if (!this.searchStatus.state.firstSearchExecuted) {
+      return Array.from({length: this.desiredCount}, () => (
+        <FacetPlaceholder numberOfValues={8} isCollapsed={this.areCollapsed} />
```

Comment:
This hardcoded `8` value will surely change once the search api documentation gets published. There will be ways to ask for a specific number of values. I put 8 since it is the default value for other facet requests.

Reply:
:+1: add a `//TODO` so we don't forget ;)